src/main/java/il/ac/bgu/cs/bp/bpjs/BPjs.java

```diff
@@ -144,5 +144,12 @@ public static void setLogDuringVerification(boolean logDuringVerification) {
         BPjs.logDuringVerification = logDuringVerification;
     }
     
+    public static void getPomVersion(){
```

Comment:
The method needs to return the version to the caller. So it needs to be of type `String`.